### PR TITLE
[CI] Oracle JDK 7 is very long dead

### PR DESCRIPTION
**Pull request by chewi**

### CODE REVIEW

**Comment by chewi** [comment]
Gah, I didn't notice that jruby-head was failing in general. I don't understand why, it's complaining about `jruby.cext.enabled` but we're not setting that as far as I can see.

**Comment by lpradovera** [comment]
We can remove some old versions, but unfortunately we do need to pass on `jruby-head`. I'll take a look too.

File: .travis.yml [hunk 1]
```diff
@@ -13,19 +13,16 @@ matrix:
     - rvm: jruby-1.7
       jdk: openjdk7
     - rvm: jruby-1.7
-      jdk: oraclejdk7
```
Comment (bklang) [inline .travis.yml]: Why leave behind the `jruby-1.7` line without a `jdk` definition? We're already testing JRuby 1.7 with openjdk7 and oraclejdk8

Comment (chewi) [reply .travis.yml]: Sorry, missed that. I won't push the change now though as it will just fail again. Let's see what @lpradovera finds out.

**Comment by lpradovera** [comment]
Looks like that value needs to be explicitly unset: https://github.com/bundler/gemstash/pull/156/files

**Comment by chewi** [comment]
In that case, they were appending to `JRUBY_OPTS`, which must have already included the option as it wasn't in the original line. I think the real reason is `before_script: export ...` doesn't work.

**Pull request merged**